prow/plugins/reopen/reopen.go

```diff
@@ -74,9 +75,19 @@ func handle(gc githubClient, log *logrus.Entry, e *github.GenericCommentEvent) e
 
 	if e.IsPR {
 		log.Info("Re-opening PR.")
-		return gc.ReopenPR(org, repo, number)
+		err := gc.ReopenPR(org, repo, number)
+		if err != nil {
```

Comment:
This will still create a comment for any error type.
This conditional and the analogous one bellow should require that the error is a `StateCannotBeChanged` error. See the [assign plugin](https://github.com/kubernetes/test-infra/blob/master/prow/plugins/assign/assign.go#L125) for an example.

Reply:
oops yes, I meant to check the error type :man_facepalming:

Reply:
done